### DISCUSSION TITLE
Fully displayed context menus can be overscrolled

### DISCUSSION THREAD

**Discussion by kadambishreyas** [ppy/osu#29472] [Q&A]
### Type

Game behaviour

### Bug description

context menu items aren't very tall. this means that i occasionally click the wrong buttons.

when the context menu is fully exposed (i.e. there are no items that need to be scrolled to), overscroll behavior is still allowed, which makes it impossible to correct my misclick by moving my mouse to the intended button before mouse-up.

i suspect this is a bug given the paradigm i described applies to context menus on both windows and macos (thus would be expected by most users, i would think).

### Screenshots or videos

https://github.com/user-attachments/assets/2ccccbbd-bfc5-4079-a6d9-eacf9a7f05af

### Version

2024.816.0-lazer

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16642861/compressed-logs.zip)

**Comment by smoogipoo** [comment]
This is intentional but I'm not sure whether to treat it as an issue or not. If it is an issue then it will need several o!f changes, including handling clicks at the menu rather than the item level.

**Reply by kadambishreyas** [reply]
sorry, perhaps i was hasty with the classification but it truly struck me odd since i don't think i've seen a menu do that (even if it is "scrollable" the scroll would normally just expand the height of the menu container on mouse-over)

**Reply by peppy** [reply]
Non-actionable unless more users have issues. I like it how it is.
